discourse_rock/rockcraft.yaml

```diff
@@ -86,14 +86,6 @@ parts:
     source: bundler
     organize:
       "*": srv/discourse/app/.bundle/
-  discourse-rad-plugin:
```

Comment:
Aren't we using this functionality in many of the documentation (example in here: https://juju.is/docs/sdk/build-a-12-factor-app-charm). Do you think we will not break it?

Reply:
It will probably break but majority doesn't use this plugin and the plugin is broken at the moment. It will create issues in Discourse upgrades that will happen soon. I opened a [PR](https://github.com/canonical/discourse-rad-plugin/pull/8) in the plugins repo to fix it but Web team said we should disable it instead.